homeassistant/components/meteoalarm/binary_sensor.py

```diff
@@ -85,10 +86,14 @@ def device_class(self):
 
     def update(self):
         """Update device state."""
+        self._attributes = {}
+        self._state = False
+
         alert = self._api.get_alert()
         if alert:
-            self._attributes = alert
-            self._state = True
-        else:
-            self._attributes = {}
-            self._state = False
+            expiration_date = datetime.datetime.fromisoformat(alert["expires"])
```

Comment:
Please use our datetime utils as much as possible, eg `parse_datetime` and `utcnow`.

https://github.com/home-assistant/core/blob/c668dcb1acd73940bc021b65ae2cafd419d21a19/homeassistant/util/dt.py#L124

https://github.com/home-assistant/core/blob/c668dcb1acd73940bc021b65ae2cafd419d21a19/homeassistant/util/dt.py#L59

Reply:
Done